cmd/telegraf/telegraf.go

```diff
@@ -36,7 +34,7 @@ var pprofAddr = flag.String("pprof-addr", "",
 	"pprof address to listen on, not activate pprof if empty")
 var fQuiet = flag.Bool("quiet", false,
 	"run in quiet mode")
-var fTest = flag.Bool("test", false, "enable test mode: gather metrics, print them out, and exit")
+var fTest = flag.Bool("test", false, "enable test mode: gather metrics, print them out, and exit. Note: Test mode only runs inputs, not processors, aggregators, or outputs")
```

Comment:
BTW, I don't think these strings are ever output.  Check `internal/usage.go`, maybe we should merge these usage files with the newly added files.

Reply:
they're supposed to output on --help.